StackoverflowChatbot/CommandProcessors/PriorityProcessor.cs

```diff
@@ -119,6 +118,7 @@ private bool TryGetNativeCommand(EventData data, out IAction? action)
 
 		private BaseCommand CreateCommandInstance(Type commandType)
 		{
+			// TODO use the service locator gdi!
```

Comment:
Completely reworked. Using service locator

Reply:
Alright. Well, I think the best way to do for me now is wait for your PR to be merged so that I can see the conflicts and resolve it myself.

Reply:
I'll help

Reply:
https://github.com/butler1233/stackoverflow-chatbot/pull/39/commits/bf884a3950f317f821ac4e2f0f1a494007e8ac70 is it correct merge?

Reply:
Yeah, merge your PR first before mine so I can pull from upstream > resolve merge conflict > push to my fork.

Reply:
I mean, have I manually merged everything from this PR to my PR? I decided to include your changes in my PR (sorry for hoax (it will be shown as I did the change)). But because of moving all stuff here and there...

Reply:
Just. Why. Would you do that? Anyway, I believe `diff` is capable enough to merge this without conflict.

Reply:
There is no `private BaseCommand CreateCommandInstance(Type commandType)` anymore in my PR. It's not clean code. First of all it's completely copied to the `Help` class. I made a `CommandFactory` which can be injected into whatever service is need. Like `Help` class.
`var command = CreateCommandInstance(commandType!);` is changed to
`var command = _commandFactory.Create(commandType!, _commandProcessor);` so the method is gone.
How do you merge that? Changing an already gone method? Also... How do you merge a `// TODO` comment into a codebase, where it's done? `CommandFactory` is a service locator.

Reply:
check out https://github.com/butler1233/stackoverflow-chatbot/pull/39/commits/fac4538f50db4b346395aa8b295793c3a4884423

Reply:
Or you could not think about any of that at all and let `git` run its course. This is essentially how resolving a merge conflict is done. You're making this an issue while it shouldn't in the first place. Let me say this again, tell Lee to accept your PR first, your changes will get merged. I will pull and merge from the `upstream` (basically this repo). Resolve the conflicts if there's any but it's highly unlikely to happen. And that's it.

Reply:
You think too much of git. Check my draft PR request into your repo. Same merge problems would occure, when you pull from upstream. You would loose a change that I pinpointed in comments there. Then just deny my pull request into your branch.